sources/postgres/infoschema.go

```diff
@@ -87,7 +103,14 @@ func (isi InfoSchemaImpl) GetRowsFromTable(conv *internal.Conv, tableId string)
 	// PostgreSQL schema and name can be arbitrary strings.
 	// Ideally we would pass schema/name as a query parameter,
 	// but PostgreSQL doesn't support this. So we quote it instead.
-	q := fmt.Sprintf(`SELECT * FROM "%s"."%s";`, conv.SrcSchema[tableId].Schema, conv.SrcSchema[tableId].Name)
+	isSchemaNamePrefixed := strings.HasPrefix(conv.SrcSchema[tableId].Name, conv.SrcSchema[tableId].Schema)
```

Comment:
Will it be better to validate prefix for schemaName + "." for our case to avoid cases were a column name might genuinely have schema name as prefix?

Reply:
Fair point! Updated!